Anytype/Sources/PresentationLayer/Modules/AllContent/AllContentViewModel.swift

```diff
@@ -138,17 +138,23 @@ final class AllContentViewModel: ObservableObject {
     }
     
     private func updateRows() {
-        if state.sort.relation.canGroupByDate {
-            let today = Date()
-            let dict = OrderedDictionary(
-                grouping: details,
-                by: { dateFormatter.localizedString(for: sortValue(for: $0) ?? today, relativeTo: today) }
-            )
+        guard state.sort.relation.canGroupByDate else  {
+            sections = [listSectionData(title: nil, details: details)]
+            return
+        }
+        
+        let today = Date()
+        let dict = OrderedDictionary(
+            grouping: details,
+            by: { dateFormatter.localizedString(for: sortValue(for: $0) ?? today, relativeTo: today) }
+        )
+        
+        if dict.count == 1 {
```

Comment:
it means when you create the new space - there is no sections there, because of only one section `Today` - is it ok?

Reply:
Yep, I think it is intended behaviour. There is no value in Today section if it is the only one